docker/autoscaler/run_autoscaler.py

```diff
@@ -28,10 +28,10 @@ def setup_logging() -> None:
 
     # Also log at DEBUG level to stdout for debugging with `kubectl logs`.
     root_logger = logging.getLogger("")
-    root_logger.setLevel(logging.DEBUG)
+    root_logger.setLevel(logging.INFO)
```

Comment:
Should probably update the comment to 
`# Also log to stdout for debugging with `kubectl logs`.`

Reply:
fixed